desktop/faqs.md

```diff
@@ -81,6 +81,19 @@ nested virtualization scenarios**. It might work in some cases, and not in other
 
 For more information, see [Running Docker Desktop in nested virtualization scenarios](../docker-for-windows/troubleshoot.md#running-docker-desktop-in-nested-virtualization-scenarios).
 
+### Docker Desktop's UI appears green, distorted, or has visual artifacts. How do I fix this?
+
+Docker Desktop uses hardware-accelerated graphics by default, which may cause problems for some GPUs. In such cases, 
+Docker Desktop will launch successfully, but some screens may appear green, distorted, 
+or have some visual artifacts.
+
+To work around this issue, disable hardware acceleration by turning on `disableHardwareAcceleration` option in `settings.json` for Docker Desktop. This file can be found in
+
+- `~/Library/Group Containers/group.com.docker/settings.json` for Mac
+- `C:\Users\{username}\AppData\Roaming\Docker\settings.json` for Windows
+ 
+and then reopen Docker Desktop's UI.
```

Comment:
Users probably don't think in terms of UI vs other components. This might be clearer:

```suggestion
Completely close and restart Docker Desktop to apply the changes.
```